webapp/src/app/shared/widgets/event-widget/event-editor/editor/event-editor.component.ts

```diff
@@ -64,17 +65,25 @@ export class EventEditorComponent implements OnChanges {
 		}
 		this.refreshAll();
 	}
-
+	
+	ngOnDestroy() {
+		EventEditorComponent.globalBase = this.base;
+	}
+	
+	show() {
+		this.detailsShown = false;
+	}
+	
```

Comment:
Maybe use getters/setters instead

```suggestion
	get base() {
		return EventEditorComponent.globalBase;
	}

	set base(value: number) {
		EventEditorComponent.globalBase = value;
	}
```

Reply:
implemented